### DISCUSSION TITLE
Issue references are very inconsistent

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#128008] [Mobile] [answered]
GitHub Mobile beta for Android

References appear with at least three styles: 

* "{Account} referenced this from {something}"
* "{Account} referenced this issue in {something}"
* "{Account} referenced this"

Steps: 
1. Open https://github.com/actions/labeler/issues/136
2. Scroll down
3. Load more

![Screenshot_20240610-080310.png](https://github.com/community/community/assets/2119212/15af47a9-5e9c-4558-829d-bba7dc0819f4)

![Screenshot_20240610-080443.png](https://github.com/community/community/assets/2119212/c5edf7cb-be6c-4bcc-ae40-0ec0154346bb)

# Expected results

1. Don't use both "in" and "from" -- pick one
2. Consistently include or omit the kind (here "issue")
3. Always include something

**Comment by Glox45** [comment]
Hi

**Comment by elisealix22** [comment, marked as answer]
Thanks again for your feedback, @jsoref ! I agree we could be more consistent here. I opened an issue to track this and we will look into it.
